peterstuart/pedals
# FFT effect

This adds a proof-of-concept effect which runs the input through an FFT, then an inverse FFT, hopefully resulting in an unchanged output.

## To Test

Use this as your pipeline.yml file:

``` yaml
effects:
  - type: Fft
```

The result should sound the same as if you used the `Transparent` effect instead.
